script.js

```diff
@@ -0,0 +1,66 @@
+// Dynamic speakers
+const speakers = [
+  {
+    image: 'img/speaker1.png',
+    name: 'Yochai Bankler',
+    occupation: 'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    description: 'Benkler studies scholars-based imrovement, and published his seminal book The Wealth in Creativity in 2006.',
+  },
+  {
+    image: 'img/speaker2.png',
+    name: 'Yochai Bankler',
+    occupation: 'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    description: 'Benkler studies scholars-based imrovement, and published his seminal book The Wealth in Creativity in 2006.',
+  },
+  {
+    image: 'img/speaker3.png',
+    name: 'Yochai Bankler',
+    occupation: 'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    description: 'Benkler studies scholars-based imrovement, and published his seminal book The Wealth in Creativity in 2006.',
+  },
+  {
+    image: 'img/speaker4.png',
+    name: 'Yochai Bankler',
+    occupation: 'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    description: 'Benkler studies scholars-based imrovement, and published his seminal book The Wealth in Creativity in 2006.',
+  },
+  {
+    image: 'img/speaker5.png',
+    name: 'Yochai Bankler',
+    occupation: 'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    description: 'Benkler studies scholars-based imrovement, and published his seminal book The Wealth in Creativity in 2006.',
+  },
+  {
+    image: 'img/speaker6.png',
+    name: 'Yochai Bankler',
+    occupation: 'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    description: 'Benkler studies scholars-based imrovement, and published his seminal book The Wealth in Creativity in 2006.',
+  },
+];
+const appendFeatured = document.getElementById('featured-speakers');
+const image = speakers.map((speak) => speak.image);
+const name = speakers.map((speak) => speak.name);
+const occupation = speakers.map((speak) => speak.occupation);
+const description = speakers.map((speak) => speak.description);
+appendFeatured.insertAdjacentHTML('afterbegin',
+  '<div class="fs-header"> <h2>Featured Speakers</h2> <hr class="hr-line"> </div>');
+
+for (let i = 0; i < speakers.length; i += 1) {
+  appendFeatured.insertAdjacentHTML('beforeend', `
+        <div class="fs-content"> 
+            <div class="speaker">
+            <img src="${image[i]}" alt="Speaker 1" class="speaker-img">
+              <div class="speaker-content">
+                  <h2 class="speaker-title">${name[i]}</h2>
+                  <p class="italic-text">
+                      ${occupation[i]}
+                  </p>
+                  <hr class="speaker-hr">
+                  <p class="speaker-description">
+                      ${description[i]}
+                  </p>
+              </div>
+            </div>
+          </div>
+      `);
```

Comment:
- [ ] Good job using JavaScript to generate dynamically the HTML for speakers, however there is one feature that you need to implement to complete this section. In the Mobile Version 📴  of your Web site , users should be about to see the `see more` button under the speakers. Make sure you have it right below 👍 

| Your design | Cindy design |
| :---: | :---: |
| ![Capture](https://github.com/Edgahkipkemoi/capstone/assets/80458199/21aad672-8fa3-41ef-b77d-8fecb2659e8b) | ![Capture1](https://github.com/Edgahkipkemoi/capstone/assets/80458199/2b3c509d-b766-427c-9ee6-dcbab9dc09fc) |